stripe/src/test/java/com/stripe/android/checkout/CheckoutViewModelTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.stripe.android.checkout
+
+import android.app.Activity
+import android.content.Intent
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.AbsFakeStripeRepository
+import com.stripe.android.ApiRequest
+import com.stripe.android.StripeRepository
+import com.stripe.android.model.ListPaymentMethodsParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.view.ActivityStarter
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.lang.IllegalStateException
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+class CheckoutViewModelTest {
+    @get:Rule
+    val rule = InstantTaskExecutorRule()
```

Comment:
@mshafrir curious if this might help with the flaky card widget tests?

Reply:
Thanks for pointing out, I'll take a look